src/server.ts

```diff
@@ -191,18 +191,32 @@ let rc_file = path.join(config.home_dir, '.analysis_rc');
 if (fs.existsSync(rc_file)) {
     let rc = JSON.parse(fs.readFileSync(rc_file, 'utf8'));
     if ('server' in rc) {
-        config.server_url = `${rc.server}/api/v1`;
+        config.server_url = `${rc.server}/api/v2`;
     }
 }
 
 let DiagnosticsEngines = [SecurityEngine];
 
-const getCAmsg = (deps, diagnostics): string => {
+const getCAmsg = (deps, diagnostics, totalCount): string => {
+    let msg = (deps.length == 1) ? `Scanned ${deps.length} runtime dependency, ` : `Scanned ${deps.length} runtime dependencies, `;
     if(diagnostics.length > 0) {
-        return `Scanned ${deps.length} runtime dependencies, flagged ${diagnostics.length} potential security vulnerabilities along with quick fixes`;
+        if (totalCount.advisoryCount > 0 && totalCount.vulnerabilityCount > 0) {
+            msg += `flagged ${totalCount.vulnerabilityCount} Known Security `;
+            msg += (totalCount.vulnerabilityCount == 1) ? `Vulnerability and ${totalCount.advisoryCount} Security ` : `Vulnerabilities and ${totalCount.advisoryCount} Security `;
+            msg += (totalCount.advisoryCount == 1) ? "Advisory along with quick fixes" : "Advisories along with quick fixes";
+        } else if (totalCount.advisoryCount == 0 && totalCount.vulnerabilityCount > 0) {
+            msg += `flagged ${totalCount.vulnerabilityCount} Known Security `; // `;
+            msg += (totalCount.vulnerabilityCount == 1) ? "Vulnerability along with quick fix" : "Vulnerabilities along with quick fixes";
+        } else if (totalCount.advisoryCount > 0 && totalCount.vulnerabilityCount == 0) {
+            msg += `flagged ${totalCount.advisoryCount} Security `;
+            msg += (totalCount.advisoryCount == 1) ? "Advisory" : "Advisories";
+        } else {
+            msg += `No potential security vulnerabilities found`;
+        }

```

Comment:
It would be great if you can simplify it.

Reply:
Sure!

Reply:
@arajkumar PTAL : https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/commit/0391ba4a038ba374107d78e1952d7d2b92367ab8